src/mscorlib/shared/System/Lazy.cs

```diff
@@ -383,7 +383,7 @@ private void PublicationOnlyViaFactory(LazyHelper initializer)
         private void PublicationOnlyWaitForOtherThreadToPublish()
         {
             var spinWait = new SpinWait();
-            while (!ReferenceEquals(_state, null))
+            while (!(_state is null))
```

Comment:
Why is this not `!= null` like the other place?

Reply:
VS defaulted to this as part of the "Null check can be simplified" fix. I will update to use `!=`